components/policy-mgt/org.wso2.carbon.policy.mgt.core/src/main/java/org/wso2/carbon/policy/mgt/core/dao/impl/FeatureDAOImpl.java

```diff
@@ -261,6 +261,28 @@ public boolean deleteFeaturesOfProfile(Profile profile) throws FeatureManagerDAO
         }
     }
 
+    @Override
+    public boolean deleteFeaturesOfProfile(int profileId) throws FeatureManagerDAOException {
```

Comment:
deleteFeaturesOfProfile does not sound like returning a boolean. BTW, do we really need to return true here if transaction is successfull?